test-suite-graalpy/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id "io.micronaut.build.internal.graal-languages-module"
```

Comment:
this should not `graal-languages-module` otherwise will publish to maven central

Reply:
resolved via https://github.com/micronaut-projects/micronaut-graal-languages/pull/5/commits/832a962e1e2f1635563fded96e2c9d333eebf4b7